src/import-data/ImportData.test.ts

```diff
@@ -353,7 +380,7 @@ describe('ImportData', () => {
           ...commonSnapshotExportQueryParams,
           snapshotId: googleSnapshotFixture.id,
         };
-        const { getWorkspaceApi, importJob, importTdr } = await setup({ queryParams });
+        const { getWorkspaceApi, importJob } = await setup({ queryParams });
 
```

Comment:
removed importTdr.... this is likely a remnant of an earlier version of the code.  Currently importTdr is an orphan jest.fn().  It is asserted below to not have been called.... which succeeds because tor orphan fn is never actually attached to anything in setup() above that would be called by code under test.  So I removed that too.